docs/tables/oci_nosql_table.md

```diff
@@ -43,3 +43,32 @@ from
 where
   cast(table_limits -> 'maxStorageInGBs' as INTEGER) > 1024;
 ```
+
+### Count child tables for parent tables with children
+
+```sql
+select
+ t2.name as parent,
+ count(t1.*) as child_count
+from
+ oci_nosql_table t1
+  join oci_nosql_table t2
+  on t1.title like t2.title || '.%'
+  and t1.title <> t2.title
+group by
+ parent;
+```
+
+### Count child tables for parent tables with and without children
+
+```sql
+select
+ t2.name as parent,
+ count(t1.*) - 1 as child_count
```

Comment:
Can we comment why we are subtracting 1?

Reply:
@rajlearner17 , Given my limited experience in query writing, there might be a more optimal way to phrase this. The deduction of 1 from the count is to ensure that the parent table is not included in the child count, as we subtract 1 to achieve this.